features/tracker-allowlist.json

```diff
@@ -395,14 +395,42 @@
                     }
                 ]
             },
-            "segment.io": {
+            "segment.com": {
                 "rules": [
+                    {
+                        "rule": "cdn.segment.com/analytics-next/bundles/130.bundle.487014ebc6f0bb448b6e.js",
```

Comment:
this seems fragile - `/130.bundle.487014ebc6f0bb448b6e` - it suggests that they are regularly updating names of those libraries . Making rule a bit broader e.g. `cdn.segment.com/analytics-next/bundles/*.js` would reduce number of rules and be less fragile IMO.

Reply:
@kdzwinel there are various requests that could be made that we don't want to match here - e.g., `segment.com/analytics-next/bundles/ajs-destination.bundle.a6950cf6bd0c8b0b0e97.js`. This should be fine as a temporary mitigation - we can review more general rules in investigations.

Reply:
Gotcha, since this is temporary I'm OK with the precise rules.